Makefile

```diff
@@ -8,6 +8,16 @@ BUILT_CUSTOM_RESOURCES=${PWD}/templates/custom-resources
 GOBIN=${PWD}/bin/tools
 COVERAGE=coverage.out
 
+DESTINATION=./bin/local/${BINARY_NAME}
+VERSION=$(shell git describe --always --tags)
+
+BINARY_S3_BUCKET_PATH=https://amazon-ecs-cli-v2.s3.amazonaws.com
+
+LINKER_FLAGS=-X github.com/aws/amazon-ecs-cli-v2/internal/pkg/version.Version=${VERSION}\
+-X github.com/aws/amazon-ecs-cli-v2/internal/pkg/cli.binaryS3BucketPath=${BINARY_S3_BUCKET_PATH}
+# RELEASE_BUILD_LINKER_FLAGS disables DWARF and symbol table generation to reduce binary size
+RELEASE_BUILD_LINKER_FLAGS=-s -w
```

Comment:
I might be wrong but from what I read it looks like `-s` should be a superset of `-w`, would you mind checking if the binary size is the same if you omit `-w`?

Reply:
`"-s -w"` provides the best results for all platforms:

# "-s -w"

22209K ecs-preview
19988K ecs-preview-amd64
16243K ecs-preview.exe

# "-s"
26393K ecs-preview
19988K ecs-preview-amd64
16243K ecs-preview.exe

# "-w"
22209K ecs-preview
22395K ecs-preview-amd64
18490K ecs-preview.exe